modules/firebase/examples_test.go

```diff
@@ -0,0 +1,41 @@
+package firebase_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go/modules/firebase"
+)
+
+func ExampleRunContainer() {
+	// runFirebaseContainer {
+	ctx := context.Background()
+
+	firebaseContainer, err := firebase.RunContainer(
+		ctx,
+		firebase.WithRoot(filepath.Join(BasePath(), "firebase")),
+	)
+	if err != nil {
+		log.Fatalf("failed to start container: %s", err)
```

Comment:
suggestion: please see the latest template for this which ensures that defer's are run by not calling `log.Fatal`, more below.

Reply:
Remade the whole PR using the new template. This code has been stale for a bit, we used it internally for over a year now :)